### PR TITLE
Fix small typo's in PHPDoc

### PR DESCRIPTION
**Pull request by holtkamp**

### CODE REVIEW

File: src/Queue.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ class Queue implements \Countable
      * The `$handler` parameter must be a valid callable that accepts your job
      * parameters, invokes the appropriate operation and returns a Promise as a
      * placeholder for its future result. If the given argument is not a valid
-     * callable, this method will reject with an `InvalidArgumentExceptionn`
+     * callable, this method will reject with an `InvalidArgumentException`
```
Comment (clue) [inline src/Queue.php]: LGTM, the README.md has the exact same typo, can you amend this? :shipit:

Comment (holtkamp) [reply src/Queue.php]: Done, my `git` skills are still too poor to do a real "amend", so you might need to squash he commits into one....

**Comment by holtkamp** [comment]
Github also offers this "squash and merge" at the tip of a button, right? 

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits

My experience is that I will fail, retry, fail again and then succeed, which will take another hour of my life... Happy to invest it, but this is a really tiny PR 👼

**Comment by clue** [comment]
@holtkamp I've reset the branch and force-pushed for you :+1: But you're missing out, you may still want to take a look at interactive rebasing and amending changes for the next time :-)

For the record: I've disabled GitHub's squash function on all my repos because it creates a new squashed commit and leaves PRs as stale branches, meh. Providing proper PRs is much cleaner and allows for a simpler git history IMHO.

Either way, now let's get this in :shipit:

**Comment by holtkamp** [comment]
>But you're missing out, you may still want to take a look at interactive rebasing and amending changes for the next time :-)

sure, believe me I did, I tried and based on your advise will try again

> For the record: I've disabled GitHub's squash function on all my repos because it creates a new squashed commit and leaves PRs as stale branches, meh. Providing proper PRs is much cleaner and allows for a simpler git history IMHO.

Ah, ok, I did not know that / see that, excusez-moi!

**Comment by clue** [comment]
@holtkamp No worries, it's a very nuanced difference and your contribution is much appreciated either way! :100:

**Pull request merged**